lib/Logger/LogStructuredParamsAllowList.h

```diff
@@ -0,0 +1,36 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2021 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Julia Casarin Puget
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include <string>
+#include <unordered_set>
+
+namespace arangodb {
+namespace structuredParams {
+const std::unordered_set<std::string> allowList = {
+    "database", "username", "url",         "collection", "queryid",
+    "viewId",   "indexid",  "replication", "pregelid",   "transactionid"};
```

Comment:
capitalization is inconsistent: `viewId` vs. `indexid`. Please use consistent capitalization, i.e. `indexId`, `pregeld`, `transactionId`, `queryId`.

Reply:
Addressed